### ISSUE TITLE
WICG maybe vanished from "relevant groups" ?

### ISSUE THREAD

**Issue by marcoscaceres** [w3c/ash-nazg#104]
I wanted to add a repo from the WICG today, but the WICG seems to have gone missing (see screenshot)... or maybe I'm bit blind 🤓. 

Also, small feature request: can you please order the "relevant groups" items alphabetically? 

![screenshot 2017-09-28 13 41 37](https://user-images.githubusercontent.com/870154/30948384-e4990902-a452-11e7-9414-894238cd179c.png)

**Comment by marcoscaceres** [comment]
Found it... it was a top 😊 But yeah... can you please sort the list.

**Comment by dontcallmedom** [comment]
the list sorted alphabetically, but first sorted by group type - ie community groups come first, working groups second. I'll make that clearer

**Comment by marcoscaceres** [comment]
I don’t know if it’s super useful to group then sort (because there are such few groups)... maybe we can distinguish them in some other way?

**Comment by dontcallmedom** [comment]
one reason it's useful is that, while there can be repos shared between groups, they can only be shared between groups of the same type (ie shared between CGs, or shared between WGs)

**Comment by marcoscaceres** [comment]
I see. So does the system know which orgs are “cg” and which are “wg”s? If so, maybe we could hide “wg” when a “cg” org is selected and vice versa.

**Comment by dontcallmedom** [comment]
no, it doesn't (and can't - e.g. there are CG repos under @w3c) - but the tool prevent selecting a CG once you've selected a WG, and vice versa

**Comment by marcoscaceres** [comment]
OK, no problem then. I guess a note about how they are organized will do.
